jest.config.js

```diff
@@ -4,6 +4,7 @@ const config = createConfig('jest', {
   setupFiles: [
     '<rootDir>/src/setupTest.js',
   ],
+  testEnvironmentOptions: { resources: 'usable' },
```

Comment:
This [tells](https://github.com/jsdom/jsdom?tab=readme-ov-file#loading-subresources) JSDom to render the content of iframes.

Reply:
Does this impact test suites' execution time?